.github/workflows/production_smoke_tests.yml

```diff
@@ -39,7 +41,7 @@ jobs:
         uses: actions/cache@v2
         with:
           path: ${{ env.pythonLocation }}
-          key: ${{ env.pythonLocation }}-${{ hashFiles('requirements.txt') }}
+          key: ${{ env.GHA_DISTRO }}-${{ env.pythonLocation }}-${{ hashFiles('requirements.txt') }}
```

Comment:
👍 for adding the OSF version so that we can switch to new ones without worrying about the cache